tests/test_neurips_conference.py

```diff
@@ -1515,13 +1515,33 @@ def test_deployment_stage(self, conference, client, helpers):
 
     def test_review_stage(self, conference, helpers, test_client, client):
 
-
         ## make submissions visible to assigned commmittee
         invitation = client.get_invitation('NeurIPS.cc/2021/Conference/-/Submission')
         invitation.reply['readers'] = {
             'values-regex': '.*'
         }
         client.post_invitation(invitation)
+
+        invitation2 = client.get_invitation('NeurIPS.cc/2021/Conference/-/Blind_Submission')
+        invitation2.replyForumViews = '''[
```

Comment:
how did you make this work? do you see the tabs in the forum page?

Reply:
Thanks for fixing this. I added this configuration using the invitation editor as well so I didn't notice this wasn't working.